recipe/meta.yaml

```diff
@@ -22,25 +22,31 @@ requirements:
     - wheel
   run:
     - python
-    - async_generator >=1.9
-    - attrs >=19.2.0
+    - attrs >=20.1.0
     - cffi >=1.14  # [win]
-    - contextvars >=2.1  # [py<37]
     - idna
     - outcome
-    - sniffio
+    - sniffio >=1.3.0
     - sortedcontainers
-    - exceptiongroup >=1.0.0rc9  # [py<311]
+    - exceptiongroup # [py<311]
 
 test:
+  source_files:
+    - pyproject.toml
```

Comment:
a comment here could be useful

Reply:
pytest configuration is in pyproject.toml.

Reply:
ok, thanks for clarifying my understanding!